README.md

```diff
@@ -108,35 +108,6 @@ Now let's talk to this node. The easiest way to do that is to use the `cockroach
 
 Check out `./cockroach help` to see all available commands.
 
-#### Building the Docker images yourself
-See [build/README.md](build/) for more information on the available Docker
-images `cockroachdb/cockroach` and `cockroachdb/cockroach-dev`.
-You can build both of these images yourself:
-
-* `cockroachdb/cockroach-dev`: `(cd build ; ./build-docker-dev.sh)`
-* `cockroachdb/cockroach`: `(cd build ; ./build-docker-deploy.sh)`
```

Comment:
I don't think building the deploy image is something we need to talk about in `README.md`.